lib/routemap.c

```diff
@@ -1463,6 +1548,46 @@ enum rmap_compile_rets route_map_add_match(struct route_map_index *index,
 	/* Add new route match rule to linked list. */
 	route_map_rule_add(&index->match_list, rule);
 
+	/* If IPv4 or IPv6 prefix-list match criteria
+	 * has been added to the route-map index, update
+	 * the route-map's prefix table.
+	 */
+	if (strncmp(match_name, IPv4_PREFIX_LIST,
```

Comment:
This compare -- strncmp(match_name ... -- seems to be used in several different places. I wonder if it's worth making this into a macro to maintain consistency if this code ever needs to be changed in the future? Not critical, just throwing the thought out there.

Reply:
Thanks, @riw777 for reviewing the changes.
It does make sense to create a macro for this purpose. I'll upload a patch with these changes.

Reply:
As requested by @riw777, I've introduced marcos in place of strncmp.